.github/workflows/run_tests.yml

```
@@ -0,0 +1,16 @@
+name: Run tests
+on:
+    push:
+jobs:
+    run_tests:
+        runs-on: ubuntu-latest
+        continue-on-error: true
+        steps:
+            -   uses: actions/checkout@v3
+            -   uses: actions/setup-java@v3.3.0
+                with:
+                    java-version: 11
+                    distribution: adopt
+            -   run: |
+                    chmod +x gradlew
+                    ./gradlew testDebug
```

We should use regular `test` here and try to fix the compile error.
```suggestion
                    ./gradlew test
```

---

Please see 770c90e and ac7fa5f.